cicd/pipeline/deploy_pipeline.sh

```diff
@@ -38,16 +38,17 @@ aws cloudformation deploy \
     --region $REGION \
     --capabilities CAPABILITY_IAM CAPABILITY_NAMED_IAM \
     --parameter-overrides \
```

Comment:
Only added the Lambda stack name param, the rest of the changes here are just putting it into alphabetical order